pkg/webhook/nodenetworkconfigurationpolicy/server.go

```diff
@@ -0,0 +1,26 @@
+package nodenetworkconfigurationpolicy
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	webhookserver "github.com/nmstate/kubernetes-nmstate/pkg/webhook/server"
+	"github.com/nmstate/kubernetes-nmstate/pkg/webhook/server/certificate"
+)
+
+const (
+	webhookName = "nmstate"
+)
+
+func Add(mgr manager.Manager) error {
+	server := webhookserver.New(mgr, webhookName, certificate.MutatingWebhook,
+		webhookserver.WithHook("/nodenetworkconfigurationpolicies-mutate", deleteConditionsHook()),
```

Comment:
I think this deserves some comments. Could you please describe why do we need two, one `delete` and one `set`? What is executed when, what do they do.